notebooks/failure_type_classifier.ipynb

```diff
@@ -0,0 +1,922 @@
+{
```

Comment:
<p>Hi @Michael, </p><p> </p><p>According to defination of new test failure given in issue#1.</p><p>new test failure: row with solid red chunk and white to the right.</p><p>We need to check for white cells towards the right. Since, new test is added and it was not ran previously. </p><p>Hence, we should except white cells towards right. </p><p> </p><p>Should we add that part to code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Good catch. So according to this definition a new test failure can only occur in the second to last position of the run length encoding list, right? </p><p> </p><p>Would removing the for loop from line 39 to 41 with the following code block satisfy the correct definition of new test failure?</p><p> </p><p><code></code><code>&nbsp;if rle[-1][0] == 0 and rle[-2][0] == -1 and rle[-2][1] &gt;= 6:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;new_test_failure.append((rle[-2][2]-rle[-2][1],rle[-2][2]))</p><p></code><code></code></p>

Reply:
<p>Yeah, that makes sense.</p><p>Another point is, we might need to ignore values 0 (not run) while calculating consecutive failures.</p><p>Like given in the issue #1. there might be not run cases in new test failures. There is one not run a cell in 6 consecutive runs in the annotated image.</p><p> </p>

Reply:
I know it's tricky. We can not ignore if there are lots of not run test cases. 
But current code will not be able to catch new test failures given in the annotated image.
If needed we can include this in the next iteration.